app/src/main/java/com/evanfuhr/pokemondatabase/activities/MainActivity.java

```diff
@@ -133,8 +147,76 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return true;
     }
 
-    void onClickMenuSetGame(MenuItem item) {
-        Toast toast = Toast.makeText(getApplicationContext(), MENU_ITEM_NOT_IMPLEMENTED_YET, Toast.LENGTH_LONG);
-        toast.show();
+    public void onClickMenuSetGame() {
+
+        // Get list of versions
+        VersionDAO versionDAO = new VersionDAO(this);
+        List<Version> versionList = versionDAO.getAllVersions();
+
+        // Get view
+        LayoutInflater layoutInflater = LayoutInflater.from(MainActivity.this);
+        View setGameVersionView = layoutInflater.inflate(R.layout.dialog_set_game_version, null);
+
+        // Setup spinner
+        Spinner versionSpinner = (Spinner) setGameVersionView.findViewById(R.id.spinner_game_version);
+        final VersionSpinnerAdapter versionSpinnerAdapter = new VersionSpinnerAdapter(this, R.layout.dialog_set_game_version, versionList);
+        versionSpinner.setAdapter(versionSpinnerAdapter);
+        // TODO: Set position
+        versionSpinner.setSelection(versionSpinnerAdapter.getPositionByVersion(_version_id));
+
+        versionSpinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+                Version version = versionSpinnerAdapter.getItem(position);
+                _version_id = version.getID();
+
+            }
+
+            @Override
+            public void onNothingSelected(AdapterView<?> parent) {
+
+            }
+        });
```

Comment:
A spinner widget is a some sort of loading indicator?

Reply:
Spinner = select

It's a drop-down. Yay Android nomenclature!